torchtune/modules/peft/_utils.py

```diff
@@ -318,6 +233,9 @@ def validate_missing_and_unexpected_for_lora(
     base_unexpected: Optional[List[str]] = None,
     lora_missing: Optional[List[str]] = None,
     lora_unexpected: Optional[List[str]] = None,
+    full_model_state_dict_keys: Optional[List[str]] = None,
+    lora_state_dict_keys: Optional[List[str]] = None,
+    base_model_state_dict_keys: Optional[List[str]] = None,
```

Comment:
We should be able to do it without these extra arguments. You should be able to leverage base_missing, base_unexpected, lora_missing, lora_unexpected in the DPO recipe to use this API as is (may just need to move the call to `validate_missing_and_unexpected_for_lora` down a few lines)